src/webui/www/private/confirmtrackerdeletion.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8">
+    <title>QBT_TR(Remove tracker)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css">
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEvent("domready", () => {
+            const url = new URI().getData("url");
+            $("confirmDeleteTrackerText").textContent = 'QBT_TR(Are you sure you want to remove tracker "%1" from all torrents?)QBT_TR[CONTEXT=TrackersFilterWidget]'.replace("%1", decodeURIComponent(url));
```

Comment:
Seems we have to use `innerHTML` instead of `textContent` as the latter will replace quotes with HTML entity.

Also it should be a good idea to sanitize the data: `window.qBittorrent.Misc.escapeHtml(decodeURIComponent(url))` in order to prevent injection attacks. The url field is from untrusted source.

Reply:
In my testing, `innerText` produced the same result as `textContent`. I had to switch to `innerHTML`.

Reply:
>I had to switch to innerHTML.

OK.

I found out it has something to do with `QBT_TR`. Without `QBT_TR`, `textContent` will show double quotes correctly.
I should investigate it later.